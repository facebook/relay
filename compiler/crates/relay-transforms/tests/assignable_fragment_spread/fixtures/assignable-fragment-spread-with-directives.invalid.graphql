# expected-to-throw
query TestQuery {
  node(id: "4") {
    ...Assignable_user @defer
  }
}

fragment Assignable_user on User @assignable {
  __typename
}