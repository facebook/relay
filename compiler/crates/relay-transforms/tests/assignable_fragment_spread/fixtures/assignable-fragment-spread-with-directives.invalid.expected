==================================== INPUT ====================================
# expected-to-throw
query TestQuery {
  node(id: "4") {
    ...Assignable_user @defer
  }
}

fragment Assignable_user on User @assignable {
  __typename
}
==================================== ERROR ====================================
✖︎ The @defer directive is not allowed on assignable fragment spreads.

  assignable-fragment-spread-with-directives.invalid.graphql:4:24
    3 │   node(id: "4") {
    4 │     ...Assignable_user @defer
      │                        ^^^^^^
    5 │   }
